code/modules/projectiles/guns/projectile/explorer_weapons.dm

```diff
@@ -0,0 +1,712 @@
+// Energy weapons for explorers.
+// For those who like pretty lights and the smell of ozone.
+
+
+//The las-rifle is the standard for energy weapons, unlike the standard laser rifle it does not have any armor penetration, \
+nor alternate firing modes. Use this as the base line for laser weapon balance. -Kazkin
+/obj/item/weapon/gun/energy/exlaserrifle
```

Comment:
from the getgo these filepaths are giving me a fucking conniption. why not just make them subtypes of the weapons theyre supposedly based off? instead of 'obj/item/weapon/gun/energy/ex(gunwhatever)', you could have just use gun/energy/(based-off)/explorer'

Reply:
I had decided early on simply to make them new weapon paths to make them easier to balance, adjust, or add new additions. In retrospect it proved unneeded as the project evolved but by then I had the bulk of it done. Functionally, all it provides now is an easier method to balance and adjust existing weapons or add new functions.